airflow-core/src/airflow/ui/src/i18n/locales/he/common.json

```
@@ -0,0 +1,79 @@
+{
+  "defaultToGraphView": "תצוגת גרף כברירת מחדל",
+  "defaultToGridView": "תצוגת רשת כברירת מחדל",
+  "logout": "התנתק",
+  "switchToDarkMode": "החלף למצב כהה",
+  "switchToLightMode": "החלף למצב בהיר",
+  "timezone": "איזור זמן",
+  "user": "משתמש",
+  "language": {
+    "chinese": "סינית מסורתית",
+    "english": "אנגלית",
+    "german": "גרמנית",
+    "korean": "קוריאנית",
+    "select": "בחר"
+  },
+  "nav": {
+    "home": "מסך הבית",
```

```suggestion
    "home": "דף הבית",
```


I think that  is more common in this context ([דף הבית](https://he.wikipedia.org/wiki/%D7%93%D7%A3_%D7%91%D7%99%D7%AA))